Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM andreiungur/ecse437:cmake-base
```

Comment:
The generic set-up commands below are very slow & are required on every run, so I packaged them into this base image [available on DockerHub](https://cloud.docker.com/repository/registry-1.docker.io/andreiungur/ecse437)

I left the (commented out) code below on purpose, to make it obvious what the base image contains.